physionet-django/events/utility.py

```diff
@@ -0,0 +1,23 @@
+import notification.utility as notification
+
+
+def notify_host_cohosts_new_registration(request, registered_user, event):
+    """
+    Notify the host and cohosts of an event of a message.
```

Comment:
Change to something like:

```
Notify the host and cohosts that a user has submitted a request to join their event.
```

Reply:
Ah yes my bad. Originally i was thinking of making this a generic function, but forgot to update the doc strings.

Thanks